platform-1/README.md

```diff
@@ -2,5 +2,27 @@
 
 ## Using the Image
 
-Image contains Blackduck client zip file under /opt
+blackduck-client is a wrapper of Synopsys Detect, in which an air-gap .zip file (containing the latest version of Detect) is stored under the **\/opt/** directory.
 
+After running this image as a container, users may extract the .zip file onto their local machine for use. However, please note that this image is not intended to be used as a persistently running container nor for running Detect from within a Docker container.
```

Comment:
For some reason I cannot open the IDETECT-4212 ticket, but if I get the gist of it, the only changes I would make in the readme would be to clarify that the only intent of the image is to extract the airgap .zip file. (Or at least that is what I believe is the intended use.)
Minor wording revision with that in mind:
"While running this image as a container, extract the `.zip` file onto your local machine for use. Note that this image is not intended to be used as a persistently running container, nor for running Detect."

Reply:
Thanks for the feedback Chris - your understanding is correct! I went ahead and made the revision!